Altis_Life.Altis/core/init.sqf

```diff
@@ -179,6 +179,16 @@ life_hideoutBuildings = [];
     false
 } count ["gang_area_1","gang_area_2","gang_area_3"];
 
+// -- Disable commanding view
+if (LIFE_SETTINGS(getNumber,"Command_View") isEqualTo 1) then {
+    addMissionEventHandler ["EachFrame",{
+        if (cameraView isEqualTo "GROUP") then {
```

Comment:
There may be an issue with using [`isEqualTo`](https://community.bistudio.com/wiki/isEqualTo) here because it performs case sensitive comparions on strings (I.e. `"GROUP"`). Safer to use [`!=`](https://community.bistudio.com/wiki/a%20!=%20b).